app/workers/stale_issue_marker.rb

```diff
@@ -0,0 +1,72 @@
+class StaleIssueMarker
+  include Sidekiq::Worker
+  sidekiq_options :queue => :miq_bot_glacial, :retry => false
+
+  include SidekiqWorkerMixin
+
+  # If an issue/pr has any of these labels, it will not be marked as stale or closed
+  PINNED_LABELS    = ['pinned']
+
+  STALE_LABEL_NAME = 'stale'
+  STALE_DATE       = 3.months.ago
```

Comment:
This is a constant, so it will take its value at the time we start the bot, not when the worker runs.  You probably should just make this is a method and call the method in `perform`